### PR TITLE
[viessmann] Store API call count in storage service

### PR DESCRIPTION
**Pull request by rogrun**
- Store API call count in storage service

Signed-off-by: Ronny Grun <ronny.grun@t-online.de>

### CODE REVIEW

File: bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/ViessmannBridgeHandler.java [hunk 1]
```diff
@@ -160,7 +165,12 @@ public void initialize() {
 
         BridgeConfiguration config = getConfigAs(BridgeConfiguration.class);
         this.config = config;
-        apiCalls = 0;
+        String storedApiCalls = this.stateStorage.get("apiCalls");
```
Comment (cweitkamp) [inline bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/ViessmannBridgeHandler.java]: Please use a const for the string `"apiCalls"`.

File: bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/ViessmannBridgeHandler.java [hunk 2]
```diff
@@ -237,6 +247,7 @@ private Integer getPollingInterval() {
 
     private void countApiCalls() {
         apiCalls++;
+        stateStorage.put("apiCalls", String.valueOf(apiCalls));
```
Comment (cweitkamp) [inline bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/ViessmannBridgeHandler.java]: Same here.

And maybe store the resulting string value in a local variable to reuse it in the next line. Omit multiple parsing actions.

**Comment by rogrun** [comment]
@cweitkamp All requested changes are done. Please review.

**Review by cweitkamp — APPROVED** [review]
LGTM. Thanks.

**Review by J-N-K — APPROVED** [review]

**Pull request merged**